components/commit/commit.js

```diff
@@ -86,3 +89,6 @@ CommitViewModel.prototype.textDiffTypeChange = function(type) {
 CommitViewModel.prototype.stopClickPropagation = function(data, event) {
   event.stopImmediatePropagation();
 }
+CommitViewModel.prototype.toggleWordWrap = function(state) {
+  this.wordWrap(state);
```

Comment:
I would either change the function to do this `this.wordWrap(!this.wordWrap());` or rename it to `wordWrapChange` (like `textDiffTypeChange`)